continuous_integration/recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }} =11
     - {{ compiler('rust') }} >=1.65.0
```

Comment:
Are we still able to specify a lower bound like thing with something like

```yaml
rust_compiler_version:
    - '>=1.65'
```

Tried this locally and things seemed to work fine, but not sure if this is just because the compiler version is invalid and getting ignored

Reply:
Think that would work

Though it could also make sense to pick a version for reproducibility reasons

Reply:
That's fair, I'll go with the latest version being picked up for now (1.69) and we can revisit if there are any issues around that.